base/response_interceptors.bal

```diff
@@ -78,3 +78,28 @@ public isolated service class FHIRResponseInterceptor {
         }
     }
 }
+
+public isolated service class FHIRResponseErrorInterceptor {
+    *http:ResponseErrorInterceptor;
+
+    isolated remote function interceptResponseError(error err) returns http:NotFound|http:BadRequest|http:InternalServerError {
+        log:printDebug("Execute: FHIR Response Error Interceptor");
+        if err is FHIRError {
```

Comment:
is the error being logged on our logs? if not shall we log the error as well?

Reply:
Internal errors and non-fhir errors are logged in the handleErrorResponse method. FHIR errors will be sent to the client as operation outcomes and they will have all the relevant information to troubleshoot issues.